### PR TITLE
Replace Email with Expiration Date

### LINKED ISSUE #38: Replace Email with Expiration Date

### PR DESCRIPTION
**Pull request by songyiang**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -81,18 +81,18 @@ public static Address parseAddress(String address) throws ParseException {
     }
 
     /**
-     * Parses a {@code String email} into an {@code Email}.
+     * Parses a {@code String expiry date} into an {@code ExpiryDate}.
```
Comment (jonahtanjz) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: Should "String expiry date" be "String expiryDate" to match the argument?

**Review by jonahtanjz — COMMENTED** [review]
Might need to change 1 JavaDoc comment, everything else looks good.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (songyiang) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: Noted. Thanks.

**Review by jonahtanjz — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/person/ExpiryDate.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a Food Item's expiry date in Simply Kitchen.
+ * Guarantees: immutable; is valid as declared in {@link #isValidExpiryDate(String)}
+ */
+public class ExpiryDate {
+
+    public static final String MESSAGE_CONSTRAINTS = "Dates should be of the format dd-MM-yyyy or dd/MM/yyyy. ";
```
Comment (rolandyuwy) [inline src/main/java/seedu/address/model/person/ExpiryDate.java]: Perhaps "Expiry dates should be of the format d-MM-yyyy, dd-MM-yyyy, d/MM/yyyy or dd/MM/yyyy." might be more accurate?

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 3]
```diff
@@ -81,18 +81,18 @@ public static Address parseAddress(String address) throws ParseException {
     }
 
     /**
-     * Parses a {@code String email} into an {@code Email}.
+     * Parses a {@code String expiryDate} into an {@code ExpiryDate}.
```
Comment (rolandyuwy) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: Very minor, should be "an" instead of "a".

**Review by rolandyuwy — COMMENTED** [review]
Very minor suggestions, otherwise looks good!

File: src/main/java/seedu/address/model/person/ExpiryDate.java (hunk 2 above)
Comment (songyiang) [reply src/main/java/seedu/address/model/person/ExpiryDate.java]: Corrected.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 3 above)
Comment (songyiang) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: Corrected.

**Pull request merged**